src/daml/Daml/Trading/Nft/CreateNftAccount.daml

```diff
@@ -0,0 +1,27 @@
+module Daml.Trading.Nft.CreateNftAccount where 
```

Comment:
1. Probably Account package
2. Probably not needed at all if you rework Account structure implementing interfaces